README.md

```diff
@@ -87,7 +98,7 @@ imports more likely.
 You'll want to import every file next, each under the appropriate account. Make sure to select Year/Month/Date as time
 format, as well as "Include transactions before account start date".
 
-Next, approve all transactions and recategorise if any did not receive a matching category on import. This shouldn't
+Next, approve all transactions and re-categorize if any did not receive a matching category on import. This shouldn't
```

Comment:
Same here, why the change?

Reply:
didn't even look at that! Used a spelling program on VScode ;) Guess it doesn't like the British spelling of things.

Categorize means to sort something into a particular group, to place something in a particular class. Categorize is the preferred North American spelling of this transitive verb, which is a verb that takes an object. Related words are categorizes, categorized, categorizing, categorization.

Categorise is the preferred British spelling, related words are categorises, categorised, categorising, categorisation. The North American spelling of categorize is also considered correct and is gaining acceptance around the world. Categorize and categorise are examples of a group of words that are spelled with a “z” in American English and with an “s” in British English.

Reply:
I know that 'categorised' is preferred British spelling, that's why I'm baffled at having it removed.